SDK_dotnet/Models/Exchange/CreateOrderRequest.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+
+namespace Gluwa.SDK_dotnet.Models.Exchange
+{
+    public sealed class CreateOrderRequest
+    {
+        /// <summary>
+        /// Conversion symbol for the order. 
+        /// </summary>
+        [Required]
+        public EConversion? Conversion { get; set; }
+
+        /// <summary>
+        /// The address that funds the source amount.
+        /// </summary>
+        [Required]
+        public string SendingAddress { get; set; }
+
+        /// <summary>
+        /// The address that the exchanged amount will be received.
+        /// </summary>
+        [Required]
+        public string ReceivingAddress { get; set; }
+
+        /// <summary>
+        /// The amount in source currency to be exchanged.
+        /// </summary>
+        [Required]
+        public string SourceAmount { get; set; }
+
+        /// <summary>
+        /// The price you want to use. Any exchange that this order will fulfill will use this price.
+        /// </summary>
+        [Required]
+        public string Price { get; set; }
+
+        public IEnumerable<ValidationResult> Validate()
```

Comment:
do you use Validate()? Remove it if you don't

Reply:
yes. This is being used in ExchangeClient.CreateOrderAsync()